### DISCUSSION TITLE
HTTP Method: Query

### DISCUSSION THREAD

**Discussion by chelodegli** [encode/httpx#2411] [Ideas] [answered]
Hi!

I'm looking for a way to use the new HTTP Method "**Query**" with httpx. Couldn't find it in the docs, so I assume it has not been implemented. Am I right? Do you know if there is going to be support for it sometime in the future?

Links:
[https://datatracker.ietf.org/doc/draft-ietf-httpbis-safe-method-w-body/](url)
[https://horovits.medium.com/http-s-new-method-for-data-apis-http-query-1ff71e6f73f3/](url)

Thanks for reading!

**Comment by lovelydinosaur** [comment, marked as answer]
Heya.
You can use `httpx.request("QUERY", ...)` for the `QUERY` method.

**Reply by chelodegli** [reply]
Thanks!!! 

Is there any expectation that new HTTP Methods might be added by default?

**Reply by lovelydinosaur** [reply]
No plans at the moment.
